Cargo.lock

```diff
@@ -1,6 +1,6 @@
 [root]
 name = "cargo-edit"
-version = "0.1.6"
+version = "0.2.0"
```

Comment:
I wonder if we can enforce bumping the version in `Cargo.lock` on every new release.